data/101/969/707/101969707.geojson

```diff
@@ -668,12 +662,10 @@
         }
     ],
     "wof:id":101969707,
-    "wof:lastmodified":1587428206,
+    "wof:lastmodified":1612402456,
     "wof:name":"S\u00e3o Paulo",
     "wof:parent_id":404550245,
     "wof:placetype":"locality",
-    "wof:population":11967825,
-    "wof:population_rank":14,
```

Comment:
I recommend instead:

`"wof:population_rank":0,`

https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/wof#population_rank

Reply:
resolved in aaccf94